src/components/formStatusFormatter/FormStatusFormatter.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../utils/variables.scss';
+
+.closedStatusContainer {
+    width: 107px;
+    height: 32px;
+    background: rgba(255, 64, 32, 0.1);
+    border-radius: 5px;
+    padding: 5px 0px 5px 15px;
+
+    span {
+        font-style: normal;
+        font-weight: $lightBold;
+        font-size: $regularSize;
+        line-height: 22px;
+        color: $red;
+    }
+
+    :nth-child(1) {
+        background-color: $red;
+        width: 8px;
+        height: 8px;
+        border-radius: 50%;
+        display: inline-block;
+        margin-right: 5px;
+    }
+}
+
+.activeStatusContainer {
+    width: 120px;
+    height: 32px;
+    background: rgba(76, 181, 118, 0.1);
+    border-radius: 5px;
+    padding: 5px 0px 5px 15px;
+
+    span {
+        font-style: normal;
```

Comment:
If you want to try something fancy here, [mixins](https://sass-lang.com/documentation/at-rules/mixin) can be created for common styles.